probe_src/libprobe/src/prov_buffer.c

```diff
@@ -37,35 +42,37 @@ int copy(const struct Path* path) {
  * We promise not to read those fields in this function.
  */
 static void prov_log_try(struct Op op) {
+#ifdef CLONE_VFORK
```

Comment:
Don't we not need this `ifdef` since `x & CLONE_VFORK` will always be 0?

Reply:
incorporated ( removed `CLONE_VFORK`definition  from `prov_buffer.c` )